app/app.py

```diff
@@ -30,9 +33,15 @@ def coconut():
     coconut_error = None
     python_error = None
 
+    # Create command to pass to subprocess
+    full_compile_args = ["coconut", filename]
+    # Append compile arguments if necessary
+    if compile_args:
+        full_compile_args += compile_args.split(' ')
```

Comment:
Let's add an else case to default to `--target sys` to support python3

Reply:
Why won't the current code result in the universal default that supports python3 anyway? http://coconut.readthedocs.io/en/master/DOCS.html#allowable-targets

Reply:
I think universal has to work on Python 2.6 and 2.7 which is why we would get a `CoconutTargetError` when using something like `async def`. Note that #1 is a refactor and takes care of this.

Reply:
Thanks, that makes sense! Done